src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,5 @@
+package duke.task;
+
+public enum TaskType {
+    Event, Deadline, Todo
```

Comment:
Since this enum is limited to the Task class, it may be better to declare it in the class to limit its scope.
Also, since enum is a group of constants, it is better to use all uppercase.